src/main/webapp/app/exercises/programming/assess/code-editor-tutor-assessment-inline-feedback.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { Feedback, FeedbackType } from 'app/entities/feedback.model';
+import { cloneDeep } from 'lodash';
+import { TranslateService } from '@ngx-translate/core';
+
+@Component({
+    selector: 'jhi-code-editor-tutor-assessment-inline-feedback',
+    templateUrl: './code-editor-tutor-assessment-inline-feedback.component.html',
+})
+export class CodeEditorTutorAssessmentInlineFeedbackComponent {
+    MANUAL = FeedbackType.MANUAL;
+    @Input()
```

Comment:
So apparently this does work, but `@Input` on the `set`ter might make more sense?